apps/omg_watcher/lib/omg_watcher/db/block.ex

```diff
@@ -26,6 +26,8 @@ defmodule OMG.Watcher.DB.Block do
     field(:hash, :binary)
     field(:eth_height, :integer)
     field(:timestamp, :integer)
+
+    timestamps(type: :utc_datetime_usec)
```

Comment:
There's a way to apply a specific timestamp format to the entire system, by setting `migration_timestamps` as repo settings and `@timestamps_opts` in each schema:

http://blog.plataformatec.com.br/2018/10/a-sneak-peek-at-ecto-3-0-breaking-changes/

Reply:
Oooh, this would be nice!
```
config :my_app, MyApp.Repo,
  migration_timestamps: [type: :timestamptz]
```

I think we can get away with adding this now since I just started adding timestamps.

Good comment!

Reply:
Since we already have `migration_timestamps` set, can’t we just leave this as simply `timestamps` (and probably define `@timestamps_opts [type: :utc_datetime_usec]` at very top?